### ISSUE TITLE
🐛 [BUG] - Wrong profile loaded on login for some characters

### ISSUE THREAD

**Issue by Tyskoll** [ClassicUO/classicuo-web#152]
### Discord Username

Jimmy the Hand

### Description

I was having issues with multiple characters + multiple profiles.
Seemingly, despite each having their own profile all except two were always loading the profile from the top of the list of profiles.
I could always load the correct one by manually doing so after login.

I poked around and found the "state" file.

`C:\Users\<user>\AppData\Local\Google\Chrome\User Data\Default\File System\000\t\30`

I opened up the most recent file and saw at the top(here, prettified and namechanges):

```
version:4,
lastUsedProfile:
{
    '3|CharacterONE':'CharacterONE_PROFILE',
    '3|Lady CharacterONE':'CharacterONE_PROFILE',
    '3|CharacterTWO':'CharacterONE_PROFILE',
    '3|Lord CharacterTWO':'CharacterTWO_PROFILE',
    '3|CharacterTHREE':'CharacterONE_PROFILE',
    '3|Lord CharacterTHREE':'CharacterTHREE_PROFILE',
    '3|CharacterFOUR':'CharacterFOUR_PROFILE',
    '3|CharacterFIVE':'CharacterFIVE_PROFILE'
} 
```

5 Characters, 8 entries 🧐

So I realized that the two that weren't affected by the bug were only listed once in this part of the file and the ones that are affected are listed twice, once with their title prefix and once without.
The ones with the prefix seem to have the correct profile associated, the ones without their title prefix seem to have the top of the list profile associated instead (because it doesn't read the prefix on login maybe?)

So basically a second entry is being created without the prefix of the character, it doesn't save the association to the in use profile? (due to having a prefix).

**Note:** part of my attempted fix was deleting default profile as it was originally the one being loaded over and over (because it was top of the list), the same thing reoccurred with the new top of the list profile.

### Reproduction steps

```bash
Reproduction by normal use with multiple  characters, a profile for each and at least one character with a fame prefix.
```

### Screenshots

```bash
![DESCRIPTION](LINK.png)
```

### Browsers

Chrome

### OS

Windows
